sdk/communication/Azure.Communication.Common/tests/CommunicationTestEnvironment.cs

```diff
@@ -25,5 +27,13 @@ public class CommunicationTestEnvironment : TestEnvironment
         public string LiveTestAccessKey => Core.ConnectionString.Parse(LiveTestConnectionString).GetRequired("accesskey");
 
         public string CommunicationTestPhoneNumber => GetVariable(AzurePhoneNumber);
+
+        public string TestPackagesEnabled => GetTestPackageEnabled();
```

Comment:
does it make sense to break the string into parts if we support multiple values like `chat|identity`? Also we can collapse the `GetTestPackageEnabled` and move its code to this property.

Reply:
btw, did you check this: https://github.com/Azure/azure-sdk-for-net/blob/850df4dc21246f6d216e8ed472313b741a8362b7/sdk/storage/Azure.Storage.Common/tests/Shared/AzuriteNUnitFixture.cs#L9

it might be a cleaner way to disable them without having to add setup to all test classes

Reply:
I checked it

> does it make sense to break the string into parts if we support multiple values like `chat|identity`? Also we can collapse the `GetTestPackageEnabled` and move its code to this property.

sorry, I don't fully understand what you mean here. can you elaborate a bit?

Reply:
> // This class is without namespace on purpose, to make sure it runs once per test assembly regardless of how tests are packaged.

I checked, but we want to skip each package, I am not so sure how we can check the env variable here and being able to check different packages accordingly